### DISCUSSION TITLE
Nuclei is not running, it says - 296 killed

### DISCUSSION THREAD

**Discussion by sherli297** [projectdiscovery/nuclei-templates#9274] [General]
Now a days I couldn't even run nuclei tool for a single domain where I enumerate subdomains and run nuclei. A complete scan is not done which says "[1]    296 killed     nuclei -l shield -eid   -nmhe |
       297 done       tee nuc_shield.txt " Same happens in VPS as well. 

moreover I exclude few templates and run the nuclei, below are I excluded templates and this is how I run the nuclei for a single domain and multiple domains

nuclei -l shield -eid wp-email-subscribers-listing,wp-woocommerce-email-verification,email-extractor,sonicwall-email-security-detect,api-abstract-email-validation,default-apache2-ubuntu-page,default-nginx-page,apache-detect,openresty-detect,basic-auth-detect,default-apache-test-all,nginx-version,HTTP-TRACE:trace-request,tech-detect:cloudflare,tech-detect:nginx,tech-detect:jsdelivr,tech-detect:google-tag-manager,tech-detect:google-font-api,tech-detect:ms-iis,http-missing-security-headers,dns-waf-detect:cloudflare,options-method,readme-md,deprecated-tls,imap-detect,openssh-detect,cname-fingerprint,cname-service:github,txt-fingerprint,caa-fingerprint,ssl-issuer,ssl-dns-names,waf-detect:shadowd,waf-detect:varnish,waf-detect:ats,waf-detect:teros,tls-version,weak-cipher-suites,pop3-detect,dmarc-detect,rdap-whois:raw,missing-sri,wildcard-tls,robots-txt-endpoint,s3-detect,favicon-detect:react,waf-detect:cloudflare,waf-detect:varnish,waf-detect:securesphere,waf-detect:nginxgeneric,waf-detect:cloudfront,waf-detect:ats,waf-detect:apachegeneric,wordpress-detect,cors-misconfig:arbitrary-origin,expired-ssl,revoked-ssl-certificate,dns-saas-service-detection:google-hosted,dns-saas-service-detection:github,dns-saas-service-detection:heroku,dns-saas-service-detection:amazon-aws,dns-saas-service-detection,dns-saas-service-detection:ghost,dns-saas-service-detection:github,nameserver-fingerprint,mx-fingerprint,mx-service-detector:Google Apps,xss-deprecated-header,google-frontend-httpserver,aws-cloudfront-service,email-extractor,aws-cloudfront-service,google-floc-disabled,jwt-token,tech-detect:google-font-api,dns-saas-service-detection:webflow,http-missing-security-headers:permissions-policy,http-missing-security-headers:x-frame-options,http-missing-security-headers:x-content-type-options,http-missing-security-headers:clear-site-data,http-missing-security-headers:cross-origin-opener-policy,http-missing-security-headers:cross-origin-resource-policy,http-missing-security-headers:strict-transport-security,http-missing-security-headers:content-security-policy,missing-sri,waf-detect:modsecurityowasp,weak-cipher-suites:tls-1.0 -nmhe | tee nuc_shield.txt

![image](https://github.com/projectdiscovery/nuclei-templates/assets/85586714/b760b4b1-a589-4a1b-b04e-7edc05b67bd5)

**Comment by olearycrew** [comment]
What version of nuclei are you running? This reminds me of an issue that was around and fixed somewhat recently

**Comment by sherli297** [comment]
v3.1.10

**Comment by olearycrew** [comment]
Thanks - since that is the latest, it must not be the issue I was thinking of that we have fixed. @tarunKoyalwar do you have any thoughts about how we could help troubleshoot this more?
